apps/web/components/v2/eventtype/AvailabilityTab.tsx

```diff
@@ -134,6 +138,7 @@ export const AvailabilityTab = () => {
             color="minimal"
             EndIcon={Icon.FiExternalLink}
             target="_blank"
+            className="whitespace-nowrap"
```

Comment:
Perhaps we could put the edit link and the timezone underneath each other on mobile? `flex-col` vs `flex-row`. Because by using `whitespace-nowrap` the text still won't conform to the grid.

Reply:
<img width="331" alt="Screenshot 2022-10-18 at 6 20 11 PM" src="https://user-images.githubusercontent.com/53316345/196434157-c9f40881-6e74-4485-969e-c648d3de8738.png">

is the ui fine ? without border doesn't look good

Reply:
I like it @Udit-takkar!

Reply:
done